packages/react-components/src/templates/DiscoverGuides.tsx

```diff
@@ -0,0 +1,106 @@
+import { css } from '@emotion/react';
+
+import { HelpSection } from '../organisms';
+import { perRem } from '../pixels';
+import { Headline2, Paragraph } from '../atoms';
+import { Accordion } from '../molecules';
+import {
+  giftIcon,
+  dashboardIcon,
+  budgetIcon,
+  reportIcon,
+  shareIcon,
+  moneyIcon,
+  toolsIcon,
+} from '../icons';
+import policyIcon from '../icons/policy';
+
+const styles = css({
+  display: 'grid',
```

Comment:
do we need a `grid`? we are not setting up row and columns, a simple flex should do the job